provider/postgis/postgis.go

```diff
@@ -369,7 +371,15 @@ func (p Provider) MVTLayer(layerName string, tile tegola.Tile, tags map[string]i
 
 	bbox := fmt.Sprintf("ST_MakeEnvelope(%v,%v,%v,%v,%v)", minPt.X(), minPt.Y(), maxPt.X(), maxPt.Y(), plyr.SRID)
 	sql := strings.Replace(plyr.SQL, BBOX, bbox, -1)
-
+	if strings.Index(sql, PIXEL_WIDTH) > -1 {
+		var res float64
+		if plyr.SRID == 4326 {
```

Comment:
Please reference the constant [here](https://github.com/terranodo/tegola/blob/41859b8d22e4224dc87a5c7bc8d3c98cfcf589dd/geometry_math.go#L5).